apps/editing-toolkit/editing-toolkit-plugin/coming-soon/coming-soon.php

```diff
@@ -47,6 +47,7 @@ function render_fallback_coming_soon_page() {
 	remove_action( 'wp_head', 'global_css', 5 );
 	remove_action( 'wp_footer', 'wpcom_subs_js' );
 	remove_action( 'wp_footer', 'stats_footer', 101 );
+	add_filter( 'infinite_scroll_archive_supported', '__return_false', 99 ); // Disable infinite scroll feature
```

Comment:
This is throwing a PHPCS error now, because it needs a full stop at the end of the comment. I think I tripped you up again 😂

Reply:
Just pushed a fix.